strongbox-client/src/main/java/org/carlspring/strongbox/client/ArtifactResolver.java

```diff
@@ -1,309 +0,0 @@
-package org.carlspring.strongbox.client;
```

Comment:
I'm not quite sure why these clients have been removed?
What am I missing?

Reply:
`ArtifactResolver` has been renamed to `RestArtifactResolver` because it internally uses `javax.ws.rs` API and whenever I entered `ArtifactResolver` I had to check what kind of tools it has been using to resolve the artifact. In my opinion `RestArtifactResolver` better describes what it actually is.

Reply:
Okay! Great, thanks! :)